canonical-websites/jaas.ai
# Fix profile loading

## Done

- Handle parsing bundles and charms without descriptions.
- Update tests to catch these errors.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open a profile (e.g. /u/hatch).
- The profile should load.
